modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm

```diff
@@ -864,6 +864,30 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 	name = "Green Pencilskirt"
 	item_path = /obj/item/clothing/under/suit/skyrat/pencil/green
 
+/datum/loadout_item/under/formal/azulea_oldblood
+	name = " Oldblood's royal regalia"
+	item_path = /obj/item/clothing/under/rank/azulean/old_blood
+	restricted_roles = list(JOB_CAPTAIN, JOB_NT_REP)
+	restricted_species = list("Akula")
+
+/datum/loadout_item/under/formal/azulea_oldblood/skirt
+	name = " Oldblood's royal regalia (skirt)"
+	item_path = /obj/item/clothing/under/rank/azulean/old_blood/skirt
+	restricted_roles = list(JOB_CAPTAIN, JOB_NT_REP)
+	restricted_species = list("Akula")
+
+/datum/loadout_item/under/formal/azulea_upstart
+	name = "Upstart's noble getup"
+	item_path = /obj/item/clothing/under/rank/azulean/upstart
+	restricted_roles = list(JOB_CAPTAIN, JOB_NT_REP)
+	restricted_species = list("Akula")
+
+/datum/loadout_item/under/formal/azulea_upstart/skirt
+	name = "Upstart's noble getup (skirt)"
```

Comment:
Also, it's just me, but we could probably lose the `'s` on all of these everywhere. Not that I particularly care, but it's just that it'd fit what we usually do elsewhere a little better (using adjectives instead, basically).

```suggestion
	name = "Upstart's Noble Getup (Skirt)"
```